app/models/provider.rb

```diff
@@ -301,6 +301,11 @@ def accredited_bodies
     end
   end
 
+  def is_it_really_really_a_scitt?
+    #is purposely named poorly to remind us that this needs to be refactored. We should be using the prgram type enum not an attribute
```

Comment:
I'm not sure this naming is a good idea, this stuff tends to spread rather than have the desired effect

Reply:
If there's further work to do I'd probably card it up and chase it being prioritised, otherwise maybe it isn't that important.

Reply:
That's the plan. Misha is currently carding it, but he wanted in named poorly to ensure we prioritise it.